config/initializers/env.rb

```diff
@@ -34,5 +34,6 @@
   blog_posts_theme_filter: ENV['ENABLE_BLOG_POSTS_THEME_FILTER'],
   csrf_without_ssl: ENV['ENABLE_CSRF_WITHOUT_SSL'],
   events_theme_filter: ENV['ENABLE_EVENTS_THEME_FILTER'],
-  search_form_autocomplete: ENV['ENABLE_SEARCH_FORM_AUTOCOMPLETE']
+  search_form_autocomplete: ENV['ENABLE_SEARCH_FORM_AUTOCOMPLETE'],
+  enable_entity_page: ENV['ENABLE_ENTITY_PAGE']
```

Comment:
The key for such a config setting should be `entity_page`, so that the full accessor is `Rails.application.config.x.enable.entity_page`